internal/service/networkcontainer/resource_network_container_test.go

```diff
@@ -36,16 +34,26 @@ func TestAccNetworkContainerRS_basicAWS(t *testing.T) {
 		CheckDestroy:             checkDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: configAWS(projectName, orgID, cidrBlock, constant.AWS, "US_EAST_1"),
+				Config: configBasic(projectID, cidrBlock, constant.AWS, "US_EAST_1"),
 				Check: resource.ComposeTestCheckFunc(
 					checkExists(resourceName),
 					resource.TestCheckResourceAttrSet(resourceName, "project_id"),
 					resource.TestCheckResourceAttr(resourceName, "provider_name", constant.AWS),
 					resource.TestCheckResourceAttrSet(resourceName, "provisioned"),
+
+					resource.TestCheckResourceAttrSet(dataSourceName, "project_id"),
+					resource.TestCheckResourceAttr(dataSourceName, "provider_name", constant.AWS),
+					resource.TestCheckResourceAttrSet(dataSourceName, "provisioned"),
+
+					resource.TestCheckResourceAttrWith(dataSourcePluralName, "results.#", acc.IntGreatThan(0)),
```

Comment:
@marcosuma this follows your idea of doing more generic tests on plural ds (like checking that there is more than 1 as opposed to to a fix number) so they can run in parallel